### ISSUE TITLE
Attention_ocr: Some pressing issues

### ISSUE THREAD

**Issue by wjbKimberly** [tensorflow/models#1884]
### System information
What is the top-level directory of the model you are using: attention_ocr
Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
OS Platform and Distribution: Linux Ubuntu 16.04
TensorFlow installed from (source or binary): binary
TensorFlow version (use command below): ('v1.0.0-65-g4763edf-dirty', '1.0.1')
CUDA/cuDNN version: 8.0

### Describe the problem
Thx a lot for your work, I have successfully run the eval.py code and get results below:
```
tensorflow/core/kernels/logging_ops.cc:79] eval/CharacterAccuracy[0.96359324]
tensorflow/core/kernels/logging_ops.cc:79] eval/SequenceAccuracy[0.83375]
```
There are some pressing issues for this project I want to ask you.

1.About functions:
　　As you have show that your model performs well on "Google Street View" datasets, could you please expose this datasets?

2.About code：
　　a. I found that there is no ".tfrecords" suffix name in your FSNS datasets, did you ommit it?
　　b. In this [snippet](https://github.com/tensorflow/models/blob/master/attention_ocr/python/datasets/fsns.py#L45), it shows that the training data has fixed width or height？
　　c. What version of checkpoint do you use in eval.py? And also, if my understanding is correct, you use 100 images and their labels as the test data to eval your model?

**Comment by alexgorban** [comment]
Answers to your questions, which better to be asked on the stackoverflow.com:
1. currently there is no plans to release the internal street view datasets
2. About code:
  a. we use the [FSNS dataset](https://github.com/tensorflow/models/tree/master/street) as is and it doesn't have the ".tfrecords" suffixes
  b. yes, input images have fixed size. For your own data you can use a different size, but all images in the dataset have to have exactly the same chosen size.
  c. for the reported number 83.79% (full sequence accuracy on the test set) we used the [provided checkpoint](http://download.tensorflow.org/models/attention_ocr_2017_05_17.tar.gz) which was created after ~400k step of training. The eval.py uses num_batches*batch_size images and their labels to do the eval. By default flags --num_batches=100, --batch_size=32 so it uses 3200 images if you run with the default arguments. To run the eval on the whole dataset you need to change is to dataset_size/batch_size.

**Comment by jart** [comment]
Closing due to inactivity.
